src/scripts/main.js

```diff
@@ -1,3 +1,94 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('tbody');
+const container = document.querySelector('.container');
+const buttonAddRow = document.querySelector('.append-row.button');
+const buttonAddColumn = document.querySelector('.append-column.button');
+const buttonRemoveColumn = document.querySelector('.remove-column.button');
+const buttonRemoveRow = document.querySelector('.remove-row.button');
+
```

Comment:
Consider adding a comment here to explain the purpose of the `updateButtonStates` function. It will help in understanding that this function manages the enabling and disabling of buttons based on the current state of the table.